main.cpp

```diff
@@ -0,0 +1,61 @@
+// assignment3.cpp : Defines the entry point for the console application.
+//
+
+
+#include <iostream>
+using std::cout;
+using std::endl;
+
+#include "shapes.h"
+#include "point.h"
+#include "shapes.cpp"
+#include "point.cpp"
```

Comment:
should never (well... with certain exceptions... but it must be done *very* carefully) include .cpp files